include/message_data_fmt.h

```diff
@@ -1,180 +1,187 @@
 #ifndef MESSAGE_DATA_FMT_H
 #define MESSAGE_DATA_FMT_H
 
-/*
- * Macros to create both a constant and a string literal from a magic value
- *   The constants are used in code files when parsing text for various purposes
- *   The strings are used in the message_data_static files themselves, as you can only concat strings with other strings
- */
-
-#ifndef GLUE
-#define GLUE(a, b) a##b
-#endif
+#include "z64save.h" // For highscores
+#include "z64item.h" // For item ids
+#include "sfx.h" // For sfx ids
 
-#define STRINGIFY(s) #s
-#define EXPAND_AND_STRINGIFY(s) STRINGIFY(s)
-
-#define HEX(N) GLUE(0x, N)
-#define STR(N) EXPAND_AND_STRINGIFY(GLUE(\x, N))
+#define TEXT_LANG_US_JP 0
+#define TEXT_LANG_EU    1
 
 /*
  * Text control characters
  */
 
-// Control character magic values, in 2-digit hex without prefix
-
-#define CTRL_NEWLINE                 01
-#define CTRL_END                     02
-#define CTRL_BOX_BREAK               04
-#define CTRL_COLOR                   05
-#define CTRL_SHIFT                   06
-#define CTRL_TEXTID                  07
-#define CTRL_QUICKTEXT_ENABLE        08
-#define CTRL_QUICKTEXT_DISABLE       09
-#define CTRL_PERSISTENT              0A
-#define CTRL_EVENT                   0B
-#define CTRL_BOX_BREAK_DELAYED       0C
-#define CTRL_AWAIT_BUTTON_PRESS      0D
-#define CTRL_FADE                    0E
-#define CTRL_NAME                    0F
-#define CTRL_OCARINA                 10
-#define CTRL_FADE2                   11
-#define CTRL_SFX                     12
-#define CTRL_ITEM_ICON               13
-#define CTRL_TEXT_SPEED              14
-#define CTRL_BACKGROUND              15
-#define CTRL_MARATHON_TIME           16
-#define CTRL_RACE_TIME               17
-#define CTRL_POINTS                  18
-#define CTRL_TOKENS                  19
-#define CTRL_UNSKIPPABLE             1A
-#define CTRL_TWO_CHOICE              1B
-#define CTRL_THREE_CHOICE            1C
-#define CTRL_FISH_INFO               1D
-#define CTRL_HIGHSCORE               1E
-#define CTRL_TIME                    1F
+// Non-Wide
```

Comment:
maybe comment here too that "non-wide" is for eng/fra/ger text and "wide" is for jpn text?